Naive greedy alg/scheduling/Swarm.py

```diff
@@ -33,6 +33,12 @@ def run_epochs(self, num_epochs, draw_stats=False, draw_graph=False):
             self._draw_graph(
                 num_epochs, pd.DataFrame([stat.to_dict() for stat in epochs_stats])
             )
+        #   save final best config
```

Comment:
This logic does not belong to the Swarm.
Applying the previous comment regarding the visualizer, you can move this logic in the main